README.md

```diff
@@ -84,16 +91,46 @@ Utilizing the CED recording interface, this project paired ecephys channels with
+`nwb-conversion-tools` verifies the integrity of all code changes by running a full test suite on short examples of real data from the formats we support. There are two classes of tests in this regard; `tests/test_internals` does not require any data to be present and represents the 'minimal' expected behavior for our package, whereas `tests/test_on_data` requires the user to both perform a full install of dependencies (`pip install -r requirements-full.txt`) as well as download the associated data for each modality. 
 
-```datalad install -rg https://gin.g-node.org/NeuralEnsemble/ephy_testing_data```
+### Install testing dependencies
+We provide two easy ways of install all the dependencies required for testing:
 
-to install the `ecephys` data, and
+1) The first is a `conda` based solution that creates an environment with all the dependencies already installed.
 
-```datalad install -rg https://gin.g-node.org/CatalystNeuro/ophys_testing_data```
+```shell
+git clone https://github.com/catalystneuro/nwb-conversion-tools
+cd nwb-conversion-tools
+conda env create -f make_env_testing.yml
+conda activate nwb_conversion_testing_env
+```
+
+Note that this will also install `datalad` which is the endorsed way of downloading the testing data plus `pytest` and `pytest-cov` which are the tools that we use on our continuous integration suit. 
 
+2) The same can be accomplished by using `pip`. In a clean environment run:
+
+```shell
+git clone https://github.com/catalystneuro/nwb-conversion-tools
+cd nwb-conversion-tools
+pip install .[test_full]]
+```
+
+Notice that this method does not install `datalad`.
+### Downloading the data
+[Datalad](https://www.datalad.org/) (`conda install datalad`) is the recommended way for downloading the data. To do this; simply call:
+
+```shell
+datalad install -rg https://gin.g-node.org/NeuralEnsemble/ephy_testing_data
+```
+to install the `ecephys` data, and
+
+```shell
+datalad install -rg https://gin.g-node.org/CatalystNeuro/ophys_testing_data
+```
 for `ophys` data.
 
+
+### Test configuration file
 Once the data is downloaded to your system, you must manually modify the [config file](https://github.com/catalystneuro/nwb-conversion-tools/blob/main/tests/test_on_data/gin_test_config.json) located in `./tests/test_on_data/gin_test_config.json` so its corresponding `LOCAL_PATH` key points to the correct folder on your system that contains the dataset folder (e.g., `ephy_testing_data` for testing `ecephys`). The code will automatically detect that the tests are being run locally, so all you need to do ensure the path is correct to your specific system.
```

Comment:
Do you think it would be beneficial to mention the default `/shared/catalystneuro/` location is a place on the DANDIHub platform (with a link to it) that everyone is able to publically access the data without having to download it themselves?